tracing-core/src/callsite.rs

```diff
@@ -515,7 +514,7 @@ mod private {
 
 #[cfg(feature = "std")]
 mod dispatchers {
-    use crate::{dispatcher, lazy::Lazy};
+    use crate::{dispatcher};
```

Comment:
```suggestion
    use crate::dispatcher;
```

Reply:
My bad, I forgot to run fmt. It's fixed in the new commit.